Automattic/jetpack
# Update Form and variations icons

Fixes #28383

## Proposed changes:
* Updates the Form block and its variations icons. (pcsBup-LM-p2)

### Other information:


## Jetpack product discussion
pcsBup-LM-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Create a post and add a Form block
* Check if the Form block and the variations icons where updated as expected
